cmd/consensus/main.go

```diff
@@ -198,6 +195,25 @@ func main() {
 				return fmt.Errorf("invalid consensus parameters: requiredApprovalsForSealConstruction > chunkAlpha")
 			}
 
+			// create the singleton instance and set the value with the flag
+			_, err := updatable_configs.AcquireRequiredApprovalsForSealConstructionSetter().SetValue(requiredApprovalsForSealConstruction)
```

Comment:
Even without generalizing the admin command, I think we can remove the global singleton and instead create a variable for the config field in this field.
```go
Module({
  requiredApprovalsForSealConstructionConfig = updatable_configs. NewRequiredApprovalsForSealConstructionConfigField(flagRequiredApprovalsForSealConstruction)
}).
...
AdminCommand({
  return &admincommon.SetRequiredApprovalsForSealingCommand{
    requiredApprovalsForSealConstructionConfig: requiredApprovalsForSealConstructionConfig,
  }
}).
...
Module({
  // create sealing engine
}).
...
Module({
  // create seal validator
}).
```